internal/backend/svc/svc.go

```diff
@@ -359,6 +360,13 @@ func makeFxOpts(cfg *Config, opts RunOptions) []fx.Option {
 				z.Info("ready", zap.String("version", version.Version), zap.String("id", fixtures.ProcessID()))
 			})
 		}),
+		fx.Invoke(func(z *zap.Logger) {
+			go func() {
+				if err := http.ListenAndServe("localhost:6060", nil); err != nil {
```

Comment:
I know 6060 is standard, but isn't it likely that it'll be bound sometimes? Maybe make the port number configurable?

Reply:
made configurable